frame/system/src/lib.rs

```diff
@@ -993,13 +993,10 @@ impl<T: Trait> Module<T> {
 	/// Deposits a log and ensures it matches the block's log data.
 	///
 	/// # <weight>
-	/// - `O(D)` where `D` length of `Digest`
-	/// - 1 storage mutation (codec `O(D)`).
+	/// - 1 storage write
```

Comment:
:+1: 
Could you also adjust the weight of the using function(s)?

Reply:
I will adjust it